sssom/io.py

```diff
@@ -120,3 +138,27 @@ def get_metadata_and_prefix_map(
                 if prefix not in prefix_map:
                     prefix_map[prefix] = uri_prefix
     return Metadata(prefix_map=prefix_map, metadata=metadata)
+
+
+def get_list_of_predicate_iri(predicate_filter: tuple) -> list:
+    """Return a list of IRIs for predicate CURIEs passed.
+
+    :param predicate_filter: CURIE OR list of CURIEs OR file path containing the same.
+    :return: A list of IRIs.
+    """
+    pred_filter_list = list(predicate_filter)
+    preds = [p for p in pred_filter_list if is_curie(p)]
+    preds_iri = [get_iri(p) for p in preds]
```

Comment:
What happens if get_iri() fails? I still think we need a way to configure get_iri() with a specific prefixmap..

Reply:
Looking at https://github.com/biopragmatics/bioregistry/blob/main/src/bioregistry/resolve_identifier.py#L249

Cany you simply pass our prefix_map in? Also set use_bioregistry_io to false, just in case.

Reply:
Done! I had to move the implementation of `get_list_of_predicate_iri()` in `io.py` instead of `cli.py` since that's where I could get the prefix_map.

Reply:
For some reason I cannot ask you to review it... so weird.

Reply:
Its because I created the branch